### ISSUE TITLE
Doesn't work when using mochify watch:true.

### ISSUE THREAD

**Issue by joshlasdin** [ferlores/mochify-istanbul#10]
When using the mochify "watch" option, the coverage reporting doesn't work correctly, it instead outputs something like the following to the console:

```
__coverage__='{"/Volumes/Server/project/client/js/app/Helpers.js":{"path":"/Volumes/Server/project/client/js/app/Helpers.js","s":{"1":1,"2":1,"3":1,"4":0},"b":{},"f":{"1":1,"2":0},"fnMap":{"1":{"name":"(anonymous_1)","line":11,"loc":{"start":{"line":11,"column":19},"end":{"line":11,"column":31}}},"2":{"name":"(anonymous_2)","line":19,"loc":{"start":{"line":19,"column":17},"end":{"line":19,"column":29}}}},"statementMap":{"1":{"start":{"line":3,"column":0},"end":{"line":4,"column":35}},"2":{"start":{"line":6,"column":0},"end":{"line":22,"column":2}},"3":{"start":{"line":12,"column":8},"end":{"line":12,"column":45}},"4":{"start":{"line":20,"column":8},"end":{"line":20,"column":68}}},"branchMap":{}},"/Volumes/Server/project/client/js/app/config.js":{"path":"/Volumes/Server/project/client/js/app/config.js","s":{"1":1,"2":1,"3":1,"4":1},"b":{"1":[1,0]},"f":{"1":1},"fnMap":{"1":{"name":"(anonymous_1)","line":21,"loc":{"start":{"line":21,"column":14},"end":{"line":21,"column":29}}}},"statementMap":{"1":{"start":{"line":3,"column":0},"end":{"line":6,"column":6}},"2":{"start":{"line":8,"column":0},"end":{"line":19,"column":3}},"3":{"start":{"line":21,"column":0},"end":{"line":23,"column":2}},"4":{"start":{"line":22,"column":4},"end":{"line":22,"column":36}}},"branchMap":{"1":{"line":5,"type":"binary-expr","locations":[{"start":{"line":5,"column":13},"end":{"line":5,"column":24}},{"start":{"line":5,"column":28},"end":{"line":5,"column":41}}]}}},"/Volumes/Server/project/client/js/app/shared/models/User.js":{"path":"/Volumes/Server/project/client/js/app/shared/models/User.js","s":{"1":1,"2":1,"3":3,"4":9,"5":9,"6":3,"7":9},"b":{"1":[9,4],"2":[3,6],"3":[9,5]},"f":{"1":3,"2":9},"fnMap":{"1":{"name":"(anonymous_1)","line":7,"loc":{"start":{"line":7,"column":16},"end":{"line":7,"column":28}}},"2":{"name":"(anonymous_2)","line":20,"loc":{"start":{"line":20,"column":17},"end":{"line":20,"column":29}}}},"statementMap":{"1":{"start":{"line":3,"column":0},"end":{"line":4,"column":35}},"2":{"start":{"line":6,"column":0},"end":{"line":31,"column":3}},"3":{"start":{"line":8,"column":8},"end":{"line":8,"column":58}},"4":{"start":{"line":21,"column":16},"end":{"line":21,"column":71}},"5":{"start":{"line":23,"column":16},"end":{"line":25,"column":17}},"6":{"start":{"line":24,"column":20},"end":{"line":24,"column":56}},"7":{"start":{"line":27,"column":16},"end":{"line":27,"column":28}}},"branchMap":{"1":{"line":21,"type":"binary-expr","locations":[{"start":{"line":21,"column":27},"end":{"line":21,"column":49}},{"start":{"line":21,"column":53},"end":{"line":21,"column":70}}]},"2":{"line":23,"type":"if","locations":[{"start":{"line":23,"column":16},"end":{"line":23,"column":16}},{"start":{"line":23,"column":16},"end":{"line":23,"column":16}}]},"3":{"line":23,"type":"binary-expr","locations":[{"start":{"line":23,"column":20},"end":{"line":23,"column":42}},{"start":{"line":23,"column":46},"end":{"line":23,"column":67}}]}}}}'
```

Any idea on a way around this?

**Comment by ferlores** [comment]
Hi there. I don't use watch in my tests in general. I've tried to set up an example with mochify without the ```mochify-istanbul``` and it is failing after the first modification. Maybe you can debug a bit your case and give memore information on what is happening

**Comment by bbqbaron** [comment]
I think that, somehow, the `after(function(){console.log(...)})` call in the `transform` function is not getting to the `instrumenter` when watched tests are being recompiled. I'm very new to stream manipulation and phantom, but I'm assuming that that call is used to provide coverage input to `writeReports`. It seems that during a watchified re-test, something about the environment is different and the `console.log` just ends up actually logging instead. Does that sound plausible? Is it helpful?

**Comment by ferlores** [comment]
Fixed 2.1.2
